docs/modules/ROOT/pages/advanced-how-tos/releasing/index.adoc

```diff
@@ -20,4 +20,9 @@ image::managed_environment.png[alt=Managed Environment]
 [4] The Managed environment team validates the initial release and configures a traffic gateway, which allows external traffic to the application.
 
 .Next steps
-Create a `releasePlan` object
\ No newline at end of file
+
+* *Create a `releasePlan` object:* The development team creates a ReleasePlan object in the developer workspace. The ReleasePlan object includes a reference to the application that the development team wants to release, along with workspace where the application is supposed to be released.
+
+* *Create a `releasePlanAdmission` object:* The Managed Environment team creates or updates the ReleasePlanAdmission object in response to the ReleasePlan object created by the development team. It indicates that the Managed Environment team has approved the application specified in the ReleasePlan object.
+
+* *Releasing an application (auto or manual):* The Managed Environment team runs the `copy-application.sh` script by defining the application name before releasing an application to production. The script copies the application and all of its components from the Development to the Managed workspace. As a result, when releasing an application to production, the Managed Environment team uses the application on its workspace rather than the one on the Developer workspace. Additionally, if the Development team adds a new component to the application on their workspace, they must notify the Managed Environment team. So they can run the `copy-application.sh` script again to ensure that the managed workspace has the most recent version of the application.
```

Comment:
I don't think this is a requirement now that SnapshotEnvironmentBindings have been removed from the services. Right @johnbieren? I don't see anything in the code that would need it.

Reply:
In that case, could you please help me with the content for *Releasing an application (auto or manual):*
@davidmogar @johnbieren